diff --git a/include/mbedtls/threading_alt.h b/include/mbedtls/threading_alt.h
new file mode 100644
index 000000000..422ed64e9
--- /dev/null
+++ b/include/mbedtls/threading_alt.h
@@ -0,0 +1,36 @@
+//this file is part of eMule
+//Copyright (C)2017-2026 Merkur ( strEmail.Format("%s@%s", "devteam", "emule-project.net") / https://www.emule-project.net )
+//
+//This program is free software; you can redistribute it and/or
+//modify it under the terms of the GNU General Public License
+//as published by the Free Software Foundation; either
+//version 2 of the License, or (at your option) any later version.
+//
+//This program is distributed in the hope that it will be useful,
+//but WITHOUT ANY WARRANTY; without even the implied warranty of
+//MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//GNU General Public License for more details.
+//
+//You should have received a copy of the GNU General Public License
+//along with this program; if not, write to the Free Software
+//Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+#pragma once
+#ifdef _MSC_VER
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+#include <winsock2.h>
+
+typedef struct
+{
+	CRITICAL_SECTION cs;
+	char is_valid;
+} mbedtls_platform_mutex_t;
+
+typedef struct
+{
+	int cond;
+} mbedtls_platform_condition_variable_t;
+
+#endif
\ No newline at end of file
diff --git a/include/psa/crypto_config.h b/include/psa/crypto_config.h
index 5c6d1d37e..789a1b78f 100644
--- a/include/psa/crypto_config.h
+++ b/include/psa/crypto_config.h
@@ -433,7 +433,7 @@
  *
  * Uncomment this to allow your own alternate threading implementation.
  */
-//#define MBEDTLS_THREADING_ALT
+#define MBEDTLS_THREADING_ALT   /* eMule: custom Windows CRITICAL_SECTION impl in threading_alt.h */
 
 /**
  * \def MBEDTLS_THREADING_PTHREAD
@@ -467,7 +467,7 @@
  *
  * Enable this layer to allow use of mutexes within Mbed TLS
  */
-//#define MBEDTLS_THREADING_C
+#define MBEDTLS_THREADING_C     /* eMule: required for THREADING_ALT */
 
 /* Memory buffer allocator options */
 //#define MBEDTLS_MEMORY_ALIGN_MULTIPLE      4 /**< Align on multiples of this value */
